k8s/k8s.go

```diff
@@ -4533,6 +4560,34 @@ func (k *k8sOps) UpdateApplicationBackup(backup *v1alpha1.ApplicationBackup) (*v
 	return k.storkClient.Stork().ApplicationBackups(backup.Namespace).Update(backup)
 }
 
+func (k *k8sOps) ValidateApplicationBackup(name, namespace string, timeout, retryInterval time.Duration) error {
```

Comment:
Also need to add `ValidateApplicationRestore`

Reply:
Fixed.